site/documentation/layouts/partials/header.html

```diff
@@ -15,15 +15,19 @@
     <link href="{{"css/featherlight.min.css" | relURL}}{{ if $assetBusting }}?{{ now.Unix }}{{ end }}" rel="stylesheet">
     <link href="{{"css/perfect-scrollbar.min.css" | relURL}}{{ if $assetBusting }}?{{ now.Unix }}{{ end }}" rel="stylesheet">
     <link href="{{"css/auto-complete.css" | relURL}}{{ if $assetBusting }}?{{ now.Unix }}{{ end }}" rel="stylesheet">
+    <link href="{{"css/atom-one-dark-reasonable.css" | relURL}}{{ if $assetBusting }}?{{ now.Unix }}{{ end }}" rel="stylesheet">
     <link href="{{"css/theme.css" | relURL}}{{ if $assetBusting }}?{{ now.Unix }}{{ end }}" rel="stylesheet">
     <link href="{{"css/hugo-theme.css" | relURL}}{{ if $assetBusting }}?{{ now.Unix }}{{ end }}" rel="stylesheet">
     {{with .Site.Params.themeVariant}}
-      <link href="{{(printf "css/theme-%s.css" .) | relURL}}{{ if $assetBusting }}?{{ now.Unix }}{{ end }}" rel="stylesheet">
+    <link href="{{(printf "css/theme-%s.css" .) | relURL}}{{ if $assetBusting }}?{{ now.Unix }}{{ end }}" rel="stylesheet">
     {{end}}
+    {{ range .Site.Params.custom_css -}}
+    <link href="{{(printf "%s" .) | relURL}}{{ if $assetBusting }}?{{ now.Unix }}{{ end }}" rel="stylesheet">
+    {{- end }}
```

Comment:
This looks like a useful new feature. We could leverage it by adding the following to the `[params]` section in `hono/site/documentation/config.toml` under :

```toml
  # Provide a list of custom css files to load relative from the `static/` folder in the site root.
  custom_css = ["css/hono.css", "css/theme-hono.css", "css/swagger-ui-3.23.11.css"]
```

Reply:
To me it looks like `theme-hono.css` is already automatically loaded by the `header.html` partial and we only want to load the swagger stuff when we are on a page that actually contains a swagger UI. It is therefore being loaded by our `custom-header.html` in that case only. We could remove the loading of `hono.css` from `custom-header.html`, though ...

Reply:
@b-abel I have added the custom_css Site param ...

Reply:
> To me it looks like `theme-hono.css` is already automatically loaded by the `header.html` partial and we only want to load the swagger stuff when we are on a page that actually contains a swagger UI. It is therefore being loaded by our `custom-header.html` in that case only. We could remove the loading of `hono.css` from `custom-header.html`, though ...

You are right. I had mainly the cache busting feature in mind and didn't look closely enough at the stylesheet loading aspect.